tests/unit/test_routing_manager.cpp

```diff
@@ -1721,6 +1721,207 @@ void TestExtPanIdChange(void)
     testFreeInstance(sInstance);
 }
 
+#if OPENTHREAD_CONFIG_SRP_SERVER_ENABLE
```

Comment:
Is this feature enabled when run in GitHub Actions?

Reply:
Good point! I submitted https://github.com/openthread/openthread/pull/8164 which adds new GH action case building and running unit test with `toranj` OT config which should cover this combinations (`SRP_SERVER` and `BORDER_ROUTING` both enabled).